include/swift/AST/DiagnosticsSema.def

```diff
@@ -3104,20 +3104,31 @@ ERROR(tuple_ellipsis,none,
       "cannot create a variadic tuple", ())
 
 WARNING(implicitly_unwrapped_optional_spelling_deprecated,none,
-        "the spelling 'ImplicitlyUnwrappedOptional' is deprecated", ())
+        "the spelling 'ImplicitlyUnwrappedOptional' in deprecated; use an explicit type followed by '!'", ())
```

Comment:
I think you still mean "_is_ deprecated"?

Reply:
Indeed, thanks.